polaris.shopify.com/src/components/Markdown/components/DoDont/index.tsx

```diff
@@ -65,25 +65,33 @@ export const DoDont = ({children}: PropsWithChildren) => {
   return (
     <Fragment>
       {prefix}
-      <div className={styles.DoDont}>
-        {dos.length ? <Do title={null}>{dos}</Do> : null}
-        {donts.length ? <Dont title={null}>{donts}</Dont> : null}
-      </div>
+      <Box className={[styles.DoDont, className]}>
```

Comment:
We're doing this so we can pass the 'margin-considered-harmful' class to the `DoDont` component wrapper where necessary. (Such as when it is a child of a `Stack` component)